config/configmap/atlas-dbaas-registry.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-atlas-registration
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+data:
+  provider: MongoDBAtlas
+  inventory_kind: MongoDBAtlasInventory
+  connection_kind: MongoDBAtlasConnection
```

Comment:
adding logo details? this is recently discussed, so I think might be a good idea to add in this PR. cc: @bmozaffa  
we can add the same format as done in the CSV
https://github.com/RHEcosystemAppEng/mongodb-atlas-kubernetes/blob/9f7e65a79d2bc82cd9a3b6239708f73d28548ced/config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml#L142-L144

Reply:
It has not been added to the ADR yet. We can add it later once it is finalized in the ADR (I added a comment there).